packages/quantic/force-app/main/default/lwc/quanticFacet/__tests__/quanticFacet.test.js

```diff
@@ -231,9 +333,674 @@ describe('c-quantic-facet', () => {
+                );
+              expect(facetSearchNoMatchMessage).not.toBeNull();
+            });
+          });
+
+          describe('when more facet search values are available', () => {
+            beforeEach(() => {
+              facetState = {
+                ...initialFacetState,
+                values: exampleFacetValues,
+                canShowMoreValues: true,
+                facetSearch: {
+                  moreValuesAvailable: true,
+                  values: exampleFacetSearchValues,
+                },
+              };
+            });
+
+            it('should display the facet search more match message', async () => {
+              const element = createTestComponent();
+              await flushPromises();
+
+              const searchboxInput = element.shadowRoot.querySelector(
+                selectors.searchboxInput
+              );
+              searchboxInput.value = 'foo';
+              await flushPromises();
+
+              const facetSearchMoreMatchMessage =
+                element.shadowRoot.querySelector(
+                  selectors.facetSearchMoreMatchMessage
+                );
+              expect(facetSearchMoreMatchMessage).not.toBeNull();
+            });
+          });
+        });
+      });
+
+      describe('when the property noSearch is set to true', () => {
+        it('should not display the facet search box input', async () => {
+          const element = createTestComponent({
+            ...defaultOptions,
+            noSearch: true,
+          });
+          await flushPromises();
+
+          const searchboxInput = element.shadowRoot.querySelector(
+            selectors.searchboxInput
+          );
+          expect(searchboxInput).toBeNull();
+        });
+
+        describe('facet values', () => {
+          it('should display the facet values', async () => {
+            const expectedFacetValues = exampleFacetValues.map(
+              (facetValue) => ({
+                ...facetValue,
+                checked: false,
+                highlightedResult: facetValue.value,
+              })
+            );
+            const element = createTestComponent({
+              ...defaultOptions,
+              noSearch: true,
+            });
+            await flushPromises();
+
+            const facetValueElements = element.shadowRoot.querySelectorAll(
+              selectors.facetValue
+            );
+            expect(facetValueElements.length).toBe(expectedFacetValues.length);
+            expectedFacetValues.forEach((facetValue, index) => {
+              expect(facetValueElements[index].item).toEqual(facetValue);
+            });
+          });
+
+          describe('the show less facet values button', () => {
+            [true, false].forEach((canShowLessValues) => {
+              it(`should ${canShowLessValues ? '' : 'not '}display show less facet values button when the state indicates that it is ${canShowLessValues ? 'possible' : 'not possible'}`, async () => {
```

Comment:
```suggestion
              it(`should ${canShowLessValues ? '' : 'not'} display show less facet values button when the state indicates that it is ${canShowLessValues ? 'possible' : 'not possible'}`, async () => {
```

Reply:
this is needed otherwise we would get `notdisplay` instead of `not display`